Demos/02-angular-connect-rest/README.md

```diff
@@ -19,7 +19,7 @@ There are many sample applications that demonstrate how to use the Microsoft Gra
 You will use a QuickStart application to demonstrate working with AngularJS and the Azure AD v2 endpoint.
```

Comment:
AngularJS is super-old. Can we update this to current version of Angular? The walkthrough in https://docs.microsoft.com/graph/tutorials/angular does this. You should be able to pretty easily point them at https://github.com/microsoftgraph/msgraph-training-angularspa/releases/latest and tell them to download the AngularQuickStart.zip file.